pkg/crc/systemd/states/state.go

```diff
@@ -9,6 +9,7 @@ type State int
 const (
 	Unknown State = iota
 	Running
+	Listening
```

Comment:
We should use a `string` instead of `iota`. It would simplify this code a lot.

`type State string`

Reply:
Not a big fan of using strings for enums, but maybe it's coming from my C background.